File: iosApp/iosApp/ui/hostroom/RoomReportAction.swift
enum RoomReportAction {
    case enter(roomId: Int64)
    case retry(roomId: Int64)
    case selectTab(tab: ReportTab)
    // 내보내기 — 텍스트 요약 네이티브 공유 (FR-063 모바일 경로)
    case clickExport
}

File: composeApp/src/commonMain/kotlin/org/sesacteamproject/passmate/ui/hostroom/RoomReportAction.kt
package org.sesacteamproject.passmate.ui.hostroom

sealed interface RoomReportAction {

    data class Enter(val roomId: Long) : RoomReportAction

    data class Retry(val roomId: Long) : RoomReportAction

    data class SelectTab(val tab: ReportTab) : RoomReportAction

    // 내보내기 — 텍스트 요약 네이티브 공유 (FR-063 모바일 경로)
    data object ClickExport : RoomReportAction
}
